### DISCUSSION TITLE
Ingesting custom zeek logs into Kibana

### DISCUSSION THREAD

**Discussion by MarkBaggett** [Security-Onion-Solutions/securityonion#2496] [Unsupported Versions] [answered]
I have a custom zeek script that is successfully generating its logs in /nsm/zeek/logs/.   I would like to consume the data into Kibana.  It is not a selectable item when running so-zeek-logs.  (hardcoded whiptail values)  After directly editing /opt/so/saltstack/local/pillar/zeeklog.sls and adding the name of my log there, then running so-zeek-restart logs are still not being ingested.   Are there additional steps I need to take to make this work?  Do you have suggestions on Filebeat logs I should be checking or other diagnostics I should do?

BTW.  WOW.  Onion 2 is amazing.  

Thanks in advance
Mark Baggett

**Comment by dougburks** [comment, marked as answer]
Hey Mark!

Have you tried restarting Filebeat with `sudo so-filebeat-restart`?

Does your new log then show up in the Filebeat config in `/opt/so/conf/filebeat/etc/filebeat.yml`?

You can find Filebeat diagnostic logs in `/opt/so/log//filebeat/`.

**Reply by MarkBaggett** [reply]
so-filebeat-restart you say?   I don't know why I didn't think to do that.   

Thanks for the help.  All fixed.

**Reply by dougburks** [reply]
Glad it worked!
